.ci/retry.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+numRetries=30
+retryDelay=10
+
+NUM=$numRetries
+until "$@"
+do
+  1>&2 echo "failure ... retrying $NUM more times"
+  sleep $retryDelay
+  ((NUM--))
```

Comment:
I am not sure, if this syntax really helps. It is correct, but not very expressive. 

I think,

let "NUM--"
or
NUM=$((NUM++))

are a bit more expressive